generators/vue/templates/src/main/webapp/app/core/jhi-footer/jhi-footer.component.ts.ejs

```diff
@@ -1,4 +1,5 @@
-import { Component, Vue } from 'vue-property-decorator';
+import { defineComponent } from 'vue';
 
-@Component
-export default class JhiFooter extends Vue {}
+export default defineComponent({
+  name: 'JhiFooter',
```

Comment:
Maybe rename the component to:
'<%=jhiPrefixCapitalized%>Footer'

Reply:
If possible I prefer to don't rename the components at big PRs.
Should the folder and file be renamed too?

Reply:
```suggestion
  name: '<%=jhiPrefixCapitalized%>Footer',
```